setup.py

```diff
@@ -67,7 +69,7 @@
           'console_scripts': ["schema-salad-tool=schema_salad.main:main", "schema-salad-doc=schema_salad.makedoc:main"]
       },
       zip_safe=True,
-      cmdclass={'egg_info': tagger},
+      # cmdclass={'egg_info': tagger},
```

Comment:
i've fixed it. 
when testing with cwltool earlier, it was giving me errors related to requirement parsing. thought commenting this out might help. forgot to uncomment.